UI/MapUI.gd

```diff
@@ -25,19 +28,31 @@ func _ready():
 	var new_room_texture_rect: TextureRect = Helpers.get_first_child_node_of_type(new_room, TextureRect)
 	var new_room_size: Vector2 = new_room_texture_rect.get_size()
 	
-	# Get the width of the floor that has the most rooms, by getting the size of what a room is w/ some offset, then 
+	# Get the width of the floor that has the most rooms, by getting the size of what a room is w/ some offset
 	var room_container_width: float = ((new_room_size.x + _padding_offset) * MapManager.map_width_array.max()) + _padding_offset
 	
+	# Check if the room_container_width doesn't meet a certain threshold. If it doesn't set it to a new width, 
+	# but save whatever the current width was for positioning later
+	var new_room_container_width: float = room_container_width
+	if (room_container_width < _MINIMUM_ROOM_WIDTH):
```

Comment:
why not just `room_container_width = min(room_container_width, _MINIMUM_ROOM_WIDTH)` instead of creating a new variable ?

Reply:
I kept the room_container_width and room_container_height mainly to help with the calculations on Line 93 and Line 97 so it can save me a calculation haha. But I'll refactor to do the calculation on those lines as well just to help with readability.